src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,23 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  const userInputToString = userInput.toString().split('');
+  const numberToGuessToString = numberToGuess.toString().split('');
```

Comment:
You are converting both 'userInput' and 'numberToGuess' to strings and then splitting them into arrays. This is a valid approach, but to make the code more concise, you could perform these operations in a single step using Array.from, like so: `Array.from(userInput.toString())`.